lib/game_page.dart

```diff
@@ -46,6 +48,19 @@ class _GamePageState extends State<GamePage> {
     settings().setRayDensity(density);
   }
 
+  final List<String> deathMessages = [
+    'YOU DIED',
+    'wasted',
+    'We\'ll get \'em next time!',
+    'Ope, you lose',
+    'We regret to inform you that you have been unalived',
+    'Character status: deceased',
+  ];
+
+  int pickDeathMessage() {
```

Comment:
you could just do `String pickDeathMessage() => deathMessages[Random().nextInt(deathMessages.length)];`

Reply:
Changed to your suggestion.